jpa/subsystem/pom.xml

```diff
@@ -95,12 +95,24 @@
 
         <dependency>
             <groupId>${project.groupId}</groupId>
-            <artifactId>wildfly-ejb3</artifactId>
+            <artifactId>wildfly-ejb3-jakarta</artifactId>
             <!-- TODO remove this version element when converting jpa subsystem to jakarta namespace -->
             <!-- This artifact should already be declared in wildfly-standard-ee-bom -->
             <version>27.0.0.Beta1</version>
         </dependency>
 
+	<dependency>
+	    <groupId>org.jboss</groupId>
+	    <artifactId>jboss-transaction-spi-jakarta</artifactId>
+	    <version>${version.org.jboss.jboss-transaction-spi}</version>
+	    <exclusions>
+	        <exclusion>
+		    <groupId>*</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
```

Comment:
Everything but the groupId and artifactId should come from dependencyManagement in the bom.

Reply:
I removed the wildcard excludes.